### DISCUSSION TITLE
Rename values by regex

### DISCUSSION THREAD

**Discussion by hasso** [grafana/grafana#38267] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

https://github.com/grafana/grafana/pull/29281 added a support for renaming metrics, the same functionality is needed for values.

**Why is this needed**:

A common scenario for us are tables with instant data – hostname values are FQDN in these, but it's too verbose, wastes a lot of space and we'd like to strip these down.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
